### PR TITLE
add missing image_flip nodes to simulation

### PR DESCRIPTION
**Pull request by fmessmer**

### CODE REVIEW

File: cob_bringup/robots/cob3-2.xml [hunk 1]
```diff
@@ -114,6 +114,7 @@
 		</include>
 		<!--include file="$(find cob_bringup)/drivers/image_flip_nodelet.launch" >
```
Comment (NadiaHG) [inline cob_bringup/robots/cob3-2.xml:115]: why nodelet with the argument `start_manager` false?

**Comment by floweisshardt** [comment]
CR: OK

File: cob_bringup/robots/cob3-9.xml [hunk 2]
```diff
@@ -157,6 +157,7 @@
 		</include>
 		<!--include file="$(find cob_bringup)/drivers/image_flip_nodelet.launch" >
```
Comment (NadiaHG) [inline cob_bringup/robots/cob3-9.xml:158]: why nodelet with the argument `start_manager` false?

**Comment by NadiaHG** [comment]
@ipa-fmw and @ipa-fxm  , what about 2 reviews are needed before merge a pull request? Also travis didn't finished

File: cob_bringup/robots/cob3-9.xml (hunk 2 above)
Comment (fmessmer) [inline cob_bringup/robots/cob3-9.xml:158]: I just kept the arguments as they were used for the real robot...there they are commented out anyway....I don't know why `start_manager` was set to `false` for cob3-2 and cob3-9...but I did not want to change that....

**Comment by fmessmer** [comment]
travis is green now :wink:

**Pull request merged**